src/main/kotlin/me/zeroeightsix/kami/feature/module/MurderMysteries.kt

```diff
@@ -27,7 +27,7 @@ object MurderMysteries : Module() {
     @Setting(name = "IgnoreWoodenShovel")
     private var ignoreWoodenShovel = false
     @Setting(name = "CooldownOnOutput") // This is how often it will tell the player in chat
-    private var updateEvery: @Constrain.Range(min = 2.0, max = 16.0, step = 0.1) Int = 10
+    private var updateEvery: @Constrain.Range(min = 2.0, max = 16.0, step = Double.MIN_VALUE) Int = 10
```

Comment:
Why was this done?

Reply:
It crashes otherwise.

Reply:
Well you should've thought about why it crashes, though it's also kind of my fault for not catching it in the first place. Of course a setting of type `Int` isn't going to accept floating point numbers - change it to a `Double` (or `Float`)